src/Discord.Net.WebSocket/SocketVoiceServer.cs

```diff
@@ -0,0 +1,19 @@
+using System.Diagnostics;
+
+namespace Discord.WebSocket
+{
+    [DebuggerDisplay(@"{DebuggerDisplay,nq}")]
+    public class SocketVoiceServer
+    {
+        public ulong GuildId { get; private set; }
+        public string Endpoint { get; private set; }
+        public string Token { get; private set; }
+
+        internal SocketVoiceServer(ulong GuildId, string Endpoint, string Token)
```

Comment:
Parameters should be camelCased, and `this` should be avoided per style consistency with the lib.